pandas/io/tests/test_stata.py

```diff
@@ -1234,6 +1233,37 @@ def test_stata_111(self):
         original = original[['y', 'x', 'w', 'z']]
         tm.assert_frame_equal(original, df)
 
+    def test_out_of_range_double(self):
+        # GH 14618
+        df = DataFrame({'ColumnOk': [0.0,
```

Comment:
can u throw some infs (and -infs) in here as well (unless that screws up the test)